Makefile

```
@@ -180,7 +194,7 @@ help:
 	@echo '  make [RUNTIME=net6] TARGETPLATFORM=unix-generic'
 	@echo
 	@echo 'to check the official mods for erroneous yaml files, run:'
-	@echo '  make [RUNTIME=net6] test'
+	@echo '  make [RUNTIME=net6] test TREAT_WARNINGS_AS_ERRORS=false'
```

```suggestion
	@echo '  make [RUNTIME=net6] TREAT_WARNINGS_AS_ERRORS=false test'
```
to match how you call it from ci.yml?

---

good catch